av1an-cli/src/lib.rs

```diff
@@ -514,10 +509,19 @@ pub fn parse_cli(args: CliOpts) -> anyhow::Result<EncodeArgs> {
     chunk_order: args.chunk_order,
     concat: args.concat,
     encoder: args.encoder,
-    extra_splits_len: if args.extra_split > 0 {
-      Some(args.extra_split)
-    } else {
-      None
+    extra_splits_len: match args.extra_split {
+      Some(x) => {
```

Comment:
This can be written with less lines like this:

```rust
match args.extra_split {
  Some(0) => None,
  Some(x) => Some(x),
  None => { /* rest of the code */ }
}
```